landing-page/landing.css

```diff
@@ -0,0 +1,566 @@
+    --MAROON-60: #2F1212;
+
+    --LIGHT-GREEN: #76C310;
+}
+
+a {
+    text-decoration: none;
+}
+
+li {
+    list-style: none;
+}
+
+/*nav-bar*/
+
+.nav-bar {
+    display: flex;
+    width: 100%;
+    height: auto;
+    background: var(--BLACK-100);
+    position: sticky;
+}
+
+
+.nav--body {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.nav--header {
+    display: inline;
+    align-items: baseline;
+    font-weight: 500;
+    font-size: 2rem;
+    padding: 1.5rem;
+}
+
+.nav--title {
+    display: inline;
+    align-items: center;
+    letter-spacing: 0.2rem;
+    font-weight: 500;
+    font-size: 3rem;
+    padding: 1.5rem;
+    color: var(--LIGHT-GREEN);
+}
+
+.nav--search {
+    display: flex;
+    gap: 2rem;
+    align-items: center;
+    justify-content: center;
+    width: 50%;
+    padding: 1rem;
+}
+
+.nav--search--input {
+    border: none;
+    width: 40rem;
+    height: 3rem;
+    padding: 2rem;
+    background: rgba(255, 255, 255, 0.11);
+    border-radius: 0.5rem;
+}
+
+.nav--search--btn {
+    border: 1px solid var(--WHITE);
+    background-color: transparent;
+    color: var(--WHITE);
+    padding: 1rem 1rem;
+    border-radius: 0.5rem;
+    font-family: 'Montserrat', sans-serif;
+}
+
+
+
+.nav--link {
+    text-decoration: none;
+    color: var(--WHITE);
+    transition: 0.5s;
+    font-weight: 500;
+    font-size: 1.5rem;
+}
+
+.link-btn {
+    border: 1px solid var(--WHITE);
+    padding: 1rem 1rem;
+    border-radius: 0.5rem;
+}
+
+.btn-solid {
+    color: var(--WHITE);
+    background-color: var(--LIGHT-GREEN);
+    border: 1px solid var(--WHITE);
+    padding: 1rem 1rem;
+    font-family: 'Montserrat', sans-serif;
+}
+
+.btn-solid:hover {
+    background-color: var(--BLACK-100);
+    color: var(--WHITE);
+    ;
+    border: 1px solid var(--WHITE);
+    padding: 1rem 1rem;
+    font-family: 'Montserrat', sans-serif;
+}
+
+.nav-links {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+}
+
+.nav--list {
+    display: flex;
+    justify-content: flex-end;
+    width: auto;
+    flex-wrap: wrap;
+    list-style: none;
+
+}
+
+.nav--item {
+    padding: 1.5rem;
+}
+
+.badge--number {
+    position: relative;
+    top: -15px;
+    right: -90%;
+    font-size: 5px;
+    font-weight: 600;
+    border-radius: 30px;
+    background-color: var(--LIGHT-GREEN);
+    color: var(--BLACK-100);
+    padding: 1px 1px;
+    border: 2px solid var(--WHITE);
+}
+
+.fa {
+    font-size: 2rem;
+    color: var(--WHITE);
+}
+
+.btn-primary {
+    background-color: gray;
+}
+
+
+/*hero-section*/
+
+.hero-section{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color: rgb(242,242,244); 
+}
+
+.hero-text {
+    width: 780px;
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 123px;
+    line-height: 162px;
+    color: #000000;
```

Comment:
be consistent with color value, use variable color naming

Reply:
ok forgot to use var here